apps/system/js/statusbar.js

```diff
@@ -259,16 +229,8 @@
           }
           break;
 
-        case 'system-resize':
-          // Reprioritize icons when:
-          // * Screen orientation changes
-          // * Software home button is enabled/disabled
-          this._updateMinimizedStatusbarWidth();
-          break;
-
         case 'sheets-gesture-end':
           this.element.classList.remove('hidden');
-          this.resumeUpdate(evt.type);
           break;
 
         case 'stackchanged':
```

Comment:
the `setAppearance()` in the `stackchanged` handler is causing a small annoyance.
From the homescreen, when warm launching the settings app the icons switch to gray during 1 frame before the statusbar get's hidden.

Remove the `setAppearance` call in `stackchanged` fixed it for me. I'm not sure if it's still useful :)